adapters/InMobi/InMobiAdapter/InMobiMediatedNativeAppInstallAd.m

```diff
@@ -0,0 +1,233 @@
+//
+//  InMobiMediatedNativeAppInstallAd.m
+//  InMobiAdapter
+//
+//  Created by Niranjan Agrawal on 1/22/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "InMobiMediatedNativeAppInstallAd.h"
+#import <GoogleMobileAds/GADMediatedNativeAdDelegate.h>
+#import "NativeAdKeys.h"
+
+@interface InMobiMediatedNativeAppInstallAd () <GADMediatedNativeAdDelegate,InMobiMediatedNativeAppInstallAdDelegate>
+
+@property(nonatomic, strong) IMNative *native;
+@property(nonatomic, copy) NSArray *mappedImages;
+@property(nonatomic, strong) GADNativeAdImage *mappedIcon;
+@property(nonatomic, copy) NSDictionary *extras;
+@property(nonatomic, strong)NSDictionary *nativeAdContentDictionary;
+
+@end
+
+@implementation InMobiMediatedNativeAppInstallAd
+
+//@synthesize adDelegate;
+@synthesize adapter = adapter_;
+//@synthesize connector = _connector;
+
+-(instancetype)initWithInMobiNativeAppInstallAd:(IMNative *)nativeAd withAdapter:(GADMAdapterInMobi*)adapter shouldDownloadImage:(BOOL)shouldDownloadImage withCache:(NSCache *)imageCache {
+    if (!nativeAd) {
+        return nil;
+    }
+    self = [super init];
+    self.adapter = adapter;
+    //self.connector = connector;
+    self.native = nativeAd;
+    NSData *data = [self.native.adContent dataUsingEncoding:NSUTF8StringEncoding];
+    NSError* error = nil;
+    shouldDownloadImage=true;
```

Comment:
Please remove `shouldDownloadImage=true;`. This causes the adapter to always download images irrespective of the publisher's preference.